src/components/layout-panel/styles/axis.module.css

```diff
@@ -1,19 +1,20 @@
-.startAxis {
-    flex-basis: 65%;
-}
-
-.endAxis {
-    flex-basis: 35%;
-    border-color: var(--colors-grey300);
-    border-style: solid;
-    border-width: 0;
-    border-inline-start-width: 1px;
-}
-
 .axisContainer {
     display: flex;
     padding-block: 2px 4px;
     padding-inline: 6px 4px;
+    background-color: white;
+}
+
+.axisContainer.columns {
+    grid-area: columns;
+}
+
+.axisContainer.rows {
+    grid-area: rows;
+}
+
+.axisContainer.filters {
+    grid-area: filters;
 }
```

Comment:
I mentioned earlier that I wasn't sure if we should have the `grid-area` definitions on the `Axis` component or if we should add wrapper elements to the `LayoutPanel` for a proper "separation of concern. I see you've added it to the `Axis` and I think this is a good call, since this is actually a sub-component, colocated with the `LayoutPanel`.